src/app/chat/views/communities.nim

```diff
@@ -418,5 +419,66 @@ QtObject:
       for chat in community.chats:
         if (chat.id == channelId):
           return chat
-      
-    
\ No newline at end of file
+
+  proc markNotificationsAsRead*(self: CommunitiesView, markAsReadProps: MarkAsReadNotificationProperties) =
+    if(markAsReadProps.communityId.len == 0 and markAsReadProps.channelId.len == 0):
+      # Remove all notifications from all communities and their channels for set types.
+
+      for t in markAsReadProps.notificationTypes:
+        case t:
+          of ActivityCenterNotificationType.NewOneToOne:
+            debugEcho("Clear all one to one notifications")
```

Comment:
We use chronicles for logs. You can see the docs here: https://github.com/status-im/nim-chronicles#logscope
It's easy to use. Here's how it would look like
```suggestion
            debug "Clear all one to one notifications"
```

Reply:
Just to be sure, you use a lot of `debugEcho`s in this file. Are those TODOs or really needed?

Reply:
@jrainville as you know, from the enum values ActivityCenterNotificationType we have an option to filter notifications only by Mention (3) and Reply (4) so far. I could solve this with a simple if condition only for Mentions, but my intention was just to sorted out a bit that part and create more general methods (markNotificationsAsRead and updateNotifications ) for handling enum values so someone who will work on these notification next, will just need to add proper handling for particular type. So actually those debug echoing are left more like a comments as they should not be executed ever.

I will update the code according to your comment and use debug instead of debugEcho.